house/views.py

```diff
@@ -68,6 +70,19 @@ def add_house(request):
     raise NotImplementedError
 
 
+def add_expense(request, house_id):
+    if request.method == "POST":
+        form = ExpenseForm(request.POST)
+        if form.is_valid():
+            house = House.objects.get(house_id=house_id)
+            Expenses.create_expense(house_name=house, amount=form.data['amount'], date=form.data['date'],
+                                    category=form.data['category'], description=form.data['description'])
+            return HttpResponseRedirect(f'/../{house_id}')
+    else:
+        form = ExpenseForm()
+    return render(request, MINE_ADD_EXPENSE_ROUTE, {'form': form})
+
```

Comment:
unnecessary extra line

Reply:
In the pycharm, if I put only one empty line, it write to me: "Expected 2 blank line"
@adiabramovitch